crates/miden-tx/src/host/tx_progress.rs

```diff
@@ -7,16 +7,32 @@ use super::{NoteId, RowIndex, TransactionMeasurements};
 
 /// Contains the information about the number of cycles for each of the transaction execution
 /// stages.
-#[derive(Clone, Default, Debug)]
+#[derive(Clone, Debug)]
 pub struct TransactionProgress {
     prologue: CycleInterval,
     notes_processing: CycleInterval,
     note_execution: Vec<(NoteId, CycleInterval)>,
     tx_script_processing: CycleInterval,
     epilogue: CycleInterval,
+    epilogue_after_compute_fee_procedure: RowIndex,
```

Comment:
Could we add a comment explaining what `epilogue_after_compute_fee_procedure` means? (other fields here a relatively self-explanatory).

Also, should we make this an `Option` to handle future cases were `compute_fee` procedure is never called?